libsel4cspace/src/cspace.c

```diff
@@ -405,6 +407,27 @@ void cspace_free_slot(cspace_t *cspace, seL4_CPtr cptr)
     }
 }
 
+seL4_Word cspace_size(cspace_t *cspace)
```

Comment:
I'm not sure this function does what it's documented to do? It seems to find the first unused slot, instead of counting the total number of used slots -- i.e. if the cspace has a lot of slots used then slot 1 is freed, I think this function would return 1 instead of `old size - 1`.